Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -319,6 +329,9 @@ export default class TimelineComponent extends Vue {
     EventBus.$on("idleLogoutWarning", function(isVisible: boolean) {
       self.idleLogoutWarning = isVisible;
     });
+
+    // To be updated with Enabled/Disabled covid message configuration
+    this.covidModal.showModal();
```

Comment:
Instead of using  the component show modal method we could set the banner property to true show-covid-modal = true